lib/cli.ex

```diff
@@ -157,6 +155,8 @@ defmodule Gradient.CLI do
 
   defp prepare_option({:stop_on_first_error, _}, opts), do: [{:crash_on_error, true} | opts]
 
+  defp prepare_option({:source_path, path}, opts), do: [{:code_path, path} | opts]
```

Comment:
Can we make this change globally in the whole project? There's 8 occurrences of `code_path`, so not a lot of work, and that way we'll maintain consistency in naming.

Reply:
done